crates/but-workspace/src/commit_engine/refs.rs

```diff
@@ -32,6 +32,7 @@ pub fn rewrite(
         == 1;
     for (old, new) in changed_commits {
         let old_git2 = old.to_git2();
+        let mut already_updated_refs: Vec<bstr::BString> = Vec::new();
```

Comment:
If one wants to name the type even though it could be inferred, this would be more common:
```suggestion
        let mut already_updated_refs = Vec::<BString>::new();
```

Reply:
Thanks, this was actually an accidental leftover and didn't intend to keep the explicit type